### ISSUE TITLE
which coco dataset version are used for ssd model evaluation?

### ISSUE THREAD

**Issue by lbingbing** [tensorflow/models#6261]
which coco dataset version are used for ssd model evaluation in the table on page https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md

Model name | Speed (ms) | COCO mAP[^1] | Outputs
ssd_resnet_50_fpn_coco  | 76 | 35 | Boxes
ssd_inception_v2_coco | 42 | 24 | Boxes

I get ssd_inception_v2_coco mAP around 27%  locally on cpu and not sure where the mismatch is from.
I'm using the following from http://cocodataset.org/#download (40504 images):

2014 Train/Val annotations [241MB]
2014 Val images [41K/6GB]

I'm using an evaluation script that is a simplified version of
https://github.com/tensorflow/tensorrt/blob/master/tftrt/examples/object_detection/object_detection.py

**Comment by rootkitchao** [comment]
I think the object detection API uses the MSCOCO2014 data set.But for evaluation,only use the coco_2014_minival dataset, which is exactly equivalent to the recently defined coco_2017_val dataset.

**Comment by lbingbing** [comment]
> I think the object detection API uses the MSCOCO2014 data set.But for evaluation,only use the coco_2014_minival dataset, which is exactly equivalent to the recently defined coco_2017_val dataset.

Thanks!
I didn't find 2014_minival on coco website and I tried coco_2014_minival from here: https://github.com/facebookresearch/Detectron/blob/master/detectron/datasets/data/README.md

It looks like 2014_minival mentioned on that page are different from val2017:
http://images.cocodataset.org/annotations/annotations_trainval2017.zip
https://dl.fbaipublicfiles.com/detectron/coco/coco_annotations_minival.tgz

I also tried 2017val from coco site (5000 images) locally. accuracy is a little different from 2014val but is still around 27%.
http://images.cocodataset.org/annotations/annotations_trainval2017.zip
http://images.cocodataset.org/zips/val2017.zip

**Comment by rootkitchao** [comment]
I found a similar problem.The SSD Mobilenet V1 FPN and the SSD Resnet V1 50 FPN were downloaded from the model zoo to run legacy/eval.py for evaluation.SSD Mobilenet V1 FPN's mAP is even higher than SSD Resnet V1 50 FPN.The mAP value obtained with the mscoco2014 minival dataset and the mscoco2017 val dataset is different.(see https://github.com/tensorflow/models/issues/6278)
I tried different tfrecord generation tools and the results did not change.Perhaps the evaluation tool in object detection has problems calculating mAP.
In addition, I have not understood why the default value of eval_config.num_examples in the configuration file of the model downloaded in the model zoo is 8000. Is the model used in the model zoo evaluation not using mscoco2014 minval?
